app/main.py

```diff
@@ -1,13 +1,45 @@
-from fastapi import FastAPI
+from fastapi import FastAPI, Request
 from fastapi.middleware.cors import CORSMiddleware
 from fastapi.middleware.gzip import GZipMiddleware
 from routers import questions, quizzes, session_answers, sessions, organizations
 from mangum import Mangum
+import random
+import string
+import time
+from logger_config import setup_logger
+
+logger = setup_logger()
 
 COMPRESS_MIN_THRESHOLD = 1000  # if more than 1000 bytes (~1KB), compress
 
 app = FastAPI()
 
+
+@app.middleware("http")
```

Comment:
why only http, i think our quiz runs on https

Reply:
https is handled by a separate layer, not this web server. this web server only sees, responds, and communicates using HTTP. Https is handled by a TLS layer which this web server doesn't have access to. The encryption and decryption of data packets is not handled by this web server.